packages/fetchai/skills/tac_control_contract/behaviours.py

```diff
@@ -282,17 +282,11 @@ def _cancel_tac(self):
     def _create_items(self) -> TransactionMessage:
```

Comment:
I think the token_ids should be an argument here so you can use the same function for both goods and game currency